app/controllers/idv/confirm_start_over_controller.rb

```diff
@@ -8,7 +8,28 @@ class ConfirmStartOverController < ApplicationController
     before_action :confirm_idv_needed
 
     def index
-      analytics.idv_gpo_confirm_start_over_visited
+      @step_indicator_step = requested_letter_before? ? :get_a_letter : :verify_phone_or_address
+
+      if request.referer == idv_request_letter_url
+        analytics.idv_gpo_confirm_start_over_before_letter_visited
+        render 'idv/confirm_start_over/before_letter'
+      else
```

Comment:
I did a double take at this initially but then saw the note in the description about how this is temporary. This is fine for now, but I don't want any `referer`-checking code sitting around long term.

Maybe just call it out a little bit:

```suggestion
     # Temporarily check referer until request letter view is updated to link to the before_letter route
      if request.referer == idv_request_letter_url
        before_letter
      else
```